cel/cel_test.go

```diff
@@ -187,7 +189,7 @@ func Test_CustomTypes(t *testing.T) {
 			},
 		},
 	}}))
-	if !out.Value().Equal(types.True).(types.Bool) {
+	if !out.Equal(types.True).(types.Bool) {
```

Comment:
If the `Equal` method can return something other than `types.Bool` (you mentioned an error), this could panic. In this case I'd recommend to use the double return and check both